src/DebuggerData.cpp

```diff
@@ -64,9 +64,8 @@ bool Breakpoints::Breakpoint::operator==(const Breakpoint &bp) const
 	return bp.condition == condition;
 }
 
-
 Breakpoints::Breakpoints()
-	: memLayout(nullptr)
+	: memLayout(nullptr), currentIndex(-1)
```

Comment:
Consider moving these kind of initialization directly in the header file. So in the header declare it as:
```int currentIndex = -1;```
But it's a matter of personal style/preference. I personally like that style because it keeps the information closer together instead of spread over two files.